tests/Dockerfile

```diff
@@ -27,4 +27,4 @@ RUN apt-get update && \
 RUN pip install --upgrade pip && \
     pip install tox
 
-CMD [ "tox","--develop" ]
+CMD [ "tox" ]
```

Comment:
Maybe simply use `[ "/bin/bash" ]` ?

Reply:
I'll leave tox here, that way `docker run iot-lab-gateway-tests` will start it, instead of starting the gateway rest server (CMD inherited from the iot-lab-gateway docker image)